python/ray/rllib/agents/ddpg/ddpg_policy_graph.py

```diff
@@ -49,25 +49,47 @@ def __init__(self,
                  high_action,
                  stochastic,
                  eps,
+                 use_gaussian_noise=False,
                  theta=0.15,
-                 sigma=0.2):
+                 sigma=0.2,
+                 target_smoothing=False,
+                 target_noise=0.2,
+                 noise_clip=0.5):
 
         # shape is [None, dim_action]
         deterministic_actions = (
             (high_action - low_action) * p_values + low_action)
 
-        exploration_sample = tf.get_variable(
-            name="ornstein_uhlenbeck",
-            dtype=tf.float32,
-            initializer=low_action.size * [.0],
-            trainable=False)
-        normal_sample = tf.random_normal(
-            shape=[low_action.size], mean=0.0, stddev=1.0)
-        exploration_value = tf.assign_add(
-            exploration_sample,
-            theta * (.0 - exploration_sample) + sigma * normal_sample)
-        stochastic_actions = deterministic_actions + eps * (
-            high_action - low_action) * exploration_value
+        if use_gaussian_noise:
+            if target_smoothing:
```

Comment:
If use_gaussian_noise is renamed to smooth_target_policy, perhaps this can be renamed to is_target?

Reply:
here, we just want to branch OU and Gaussian and, in the Gaussian condition, we also have to distinguish act from target act where `is_target` is an appropriate name, imo.